src/influencetx/finances/models.py

```diff
@@ -36,6 +43,14 @@ class Job(models.Model):
     financial_disclosure = models.ForeignKey(FinancialDisclosure,
                                              on_delete=models.CASCADE,
                                              related_name='jobs')
+    job_type = models.ForeignKey(JobType, on_delete=models.SET_NULL,
+                                 blank=True, null=True, related_name='jobs')
```

Comment:
Might be clearer to label this as "types", so you fetch "jobs" with related "types" in the views.